<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NVKMainWindow</class>
 <widget class="QMainWindow" name="NVKMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>713</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,12" columnstretch="1,4">
    <item row="0" column="0" colspan="2">
     <widget class="UserPanelView" name="userPanelView">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,7">
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pageSizeLabel">
          <property name="text">
           <string>Page size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="productsInCategoryLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="CategoriesView" name="categoriesView">
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ProductsView" name="productsView">
        <property name="backgroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>59</red>
           <green>53</green>
           <blue>52</blue>
          </color>
         </brush>
        </property>
        <property name="sceneRect">
         <rectf>
          <x>0.000000000000000</x>
          <y>0.000000000000000</y>
          <width>0.000000000000000</width>
          <height>0.000000000000000</height>
         </rectf>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="previousPageButton">
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPageButton">
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProductsView</class>
   <extends>QGraphicsView</extends>
   <header>frontend/views/ProductsView.h</header>
  </customwidget>
  <customwidget>
   <class>CategoriesView</class>
   <extends>QGraphicsView</extends>
   <header>frontend/views/CategoriesView.h</header>
  </customwidget>
  <customwidget>
   <class>UserPanelView</class>
   <extends>QGraphicsView</extends>
   <header>frontend/views/UserPanelView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
